docs/.vitepress/theme/styles/fixes.scss

```diff
@@ -526,14 +526,6 @@ h2 + .custom-block {
   transition-duration: 0.1s;
 }
 
-.vp-doc .custom-block code {
-  color: var(--vp-c-text-1);
-  box-shadow: 0 0 0 1px rgba(0,0,0,0.1);
-  :root.dark & {
-    box-shadow: 0 0 0 1px rgba(255,255,255,0.16);
-  }
```

Comment:
I removed this code because it causes a weird issue for code blocks inside warnings / tips.

For instance in https://webcontainers.io/guides/working-with-the-file-system, we've got:

![weird lines](https://github.com/stackblitz/webcontainer-docs/assets/4008156/a4216f3f-3421-45ad-a1cd-55108a3d7ab6)